HybrasylXML/XSD/Castables.xsd

```diff
@@ -413,9 +413,9 @@
   <xs:complexType name="AddStatus">
     <xs:simpleContent>
       <xs:extension base="hyb:String8">
-        <xs:attribute name="Duration" use="optional" default="0" type="xs:int"/>
-        <xs:attribute name="Intensity" use="optional" default="1.0" type="xs:float"/>
-        <xs:attribute name="Speed" default="1.0" use="optional" type="xs:float"/>
+        <xs:attribute name="Duration"  default="0" type="xs:int"/>
+        <xs:attribute name="Intensity" default="1.0" type="xs:float"/>
+        <xs:attribute name="Speed" default="1.0" type="xs:float"/>
```

Comment:
These should be optional, and not required.

Reply:
Optional with a default value has the same effect.

Reply:
Yes, except optional was removed.

Reply:
Right, because in my mind if a default results in a value being set, `optional` is unclear / unneeded. Since it has no effect on the generated code (optional for instance doesn't mean it is a `float?`) it seems spurious to me. I can add it back though if you feel it is clearer.